dnf-behave-tests/features/repo-sync-errors.feature

```diff
@@ -0,0 +1,126 @@
+Feature: Tests for error messages when failed to download repository metadata
```

Comment:
It seemed to me that the ``repo-sync.feature`` primarily tests the behaviour with the different configuration options, whereas I want to test the various error messages. But the last test in ``repo-sync.feature`` fits more into the second category, too, and I don't mind adding it all together if you think it would be better.

Reply:
I think they could go into the single file (`repo-sync.feature`). They all test error handling for the most part, just some of them are misconfiguration and some are a bit different.

It also seems some of the scenarios are duplicit to the ones in `repo-sync.feature` - they seem to test the same things.